test/e2e/helpers.js

```diff
@@ -769,7 +769,13 @@ async function validateContractDetails(driver) {
   await driver.clickElement({ text: 'Got it', tag: 'button' });
 
   // Approve signing typed data
-  await driver.clickElement('[data-testid="signature-request-scroll-button"]');
+  try {
+    await driver.clickElement(
+      '[data-testid="signature-request-scroll-button"]',
+    );
+  } catch (error) {
+    // Ignore error if scroll button is not present
```

Comment:
why are we ignoring?

Reply:
Some elements in the UI are different between Stable and Flask: on Flask there is a warning (IIRC) that takes more space and makes a scroll button appear; on stable, there is not such problem.

Reply:
Gotcha.